modules/ppcp-agentic-commerce/src/Helper/AgenticCheckoutProcessor.php

```diff
@@ -145,7 +144,7 @@ private function create_order( PayPalOrder $paypal_order, $cart_data, PayPalCart
 	/**
 	 * Build payer data from PayPal cart.
 	 *
-	 * @param PayPalCart $cart The PayPal cart with customer data.
+	 * @param PayPalCart $cart The PayPal cart.
 	 * @return array Payer data array.
 	 */
 	private function build_payer_data( PayPalCart $cart ): array {
```

Comment:
Some inconsistencies with the return types in this file. Some are defined ... like `:array ` but some have missing return types